virttest/utils_net.py

```diff
@@ -2831,6 +2831,17 @@ def __init__(self, params, vm_name):
                 except IndexError:
                     existing_value = None
                 nic_dict[propertea] = nic_params.get(propertea, existing_value)
+                if propertea == "netdst" and existing_value is not None:
+                    if existing_value.split(':', 1)[0] == "shell":
+                        existing_value = decode_to_text(process.system_output(
+                            existing_value.split(':', 1)[1], shell=True))
+                        if existing_value:
+                            nic_dict[propertea] = existing_value
+                        else:
+                            raise exceptions.TestError(
+                                "netdst is null,please check the shell command")
+                    else:
+                        nic_dict[propertea] = existing_value
```

Comment:
This change seems meaningless, it has nothing different with your original code. Previously even `if propertea == "netdst" and existing_value is not None`, `nic_dict[propertea]` should equals to `nic_params[propertea]` when it exists, you still change it.

Reply:
@vivianQizhu , I reassign the code and will not touch "existing_value ", could you help review , the test results has been updated in bz.